cssproject.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title> Delicious</title>
+		<link rel="stylesheet" href=normalize.css>
+			<style>
+			body {
+				font-family: Times, serif;
+				font-size: 12px;
+				padding-left: 170px;}
+				h1{
+					line-height: 40px;
+					font-family: "Monotype Corsiva",cursive;
+					font-size: 60px;
+					margin-bottom: 1px;}
+				h2{
+					font-family: courier, monospace;
+					line-height: 0px;
+					font-weight: lighter;
+					font-size: 12px;
+					padding-left: 50px;}
+				h3{
+					font-family: arial, sans-serif;
+					font-size: 45px;
+					margin-bottom: 1px;
+				}
+				h4{
+					font-weight: lighter;
+					line-height: : 0px;
+					margin-top: 0px;
+					margin-bottom: 20px;
+				}
+				p.footer{
+					padding-left: 80px;
+					font-size: 15px;
+				}
+				p.steps1{
+					text-indent: 20px;
+					font-size: 15px;
+				}
+				p.steps2{
+					text-indent: 20px;
+					font-size: 15px;
+				}
+				p.steps3{
+					text-indent: 20px;
+					font-size: 15px;		
+				}
+				p.checklist{
+					font-family: courier;
+					background-image: url("/home/jon/CodeFellows/CSSproject/list-bg.png")
+					font-size: 15px;
+				}
+				p.topdirectbar{
+					padding-right: 310px;
+					text-align: right;
+				}
+				p.printico{
+					padding-right: 340px;
+					text-align: right
+				}
+				p.imagevan{
+					text-align: left;
+				}
+				p.imagesmall{
+					text-align: center;
+				}
+				}
+			</style>
+	</head>
+	<body>
+		
+		<h1><img src="logo.png" href="logo" align="top">Delicious<br /><h2>THE BEST FOOD BLOG ON THE WEB.</h2></h1>
```

Comment:
Improper use of the h1 tag. There really only should be test in heading tags. Also, don't use the br tag to achieve line breaks, use CSS.